pkg/xds/envoy/clusters/health_check_configurer.go

```diff
@@ -25,9 +27,32 @@ func (e *healthCheckConfigurer) Configure(cluster *envoy_api.Cluster) error {
 		return nil
 	}
 	activeChecks := e.healthCheck.Spec.Conf
+	tcpHealthCheck := &envoy_core.HealthCheck_TcpHealthCheck{}
+
+	if activeChecks.Tcp != nil {
+		if activeChecks.Tcp.Send != nil {
+			tcpHealthCheck.Send = &envoy_core.HealthCheck_Payload{
+				Payload: &envoy_core.HealthCheck_Payload_Text{
+					Text: hex.EncodeToString(activeChecks.Tcp.Send.Value),
+				},
+			}
+		}
+		if activeChecks.Tcp.Receive != nil {
+			var receive []*envoy_core.HealthCheck_Payload
+			for _, r := range activeChecks.Tcp.Receive {
+				receive = append(receive, &envoy_core.HealthCheck_Payload{
+					Payload: &envoy_core.HealthCheck_Payload_Text{
+						Text: hex.EncodeToString(r.Value),
+					},
+				})
+			}
+			tcpHealthCheck.Receive = receive
+		}
+	}
+
```

Comment:
Shoudl we add an entry in the testing:
pkg/xds/envoy/clusters/health_check_configurer_test.go

Reply:
What kind of entry you mean?
Aren't the tests I added enough?

Reply:
@nickolaev could you look again and say if we can resolve this comment or you think I should are anything else?